tests/wpt/webgl/meta/conformance/textures/misc/tex-image-and-sub-image-2d-with-array-buffer-view.html.ini

```diff
@@ -0,0 +1,6 @@
+[tex-image-and-sub-image-2d-with-array-buffer-view.html]
+  expected:
+    if os == "linux": TIMEOUT
+  [Overall test]
+    expected:
+      if os == "linux": NOTRUN
```

Comment:
is it an intermittent PASS or an intermittent TIMEOUT 👀?

```
  ▶ OK [expected TIMEOUT] /_webgl/conformance/textures/misc/tex-image-and-sub-image-2d-with-array-buffer-view.html
  ▶ Unexpected subtest result in /_webgl/conformance/textures/misc/tex-image-and-sub-image-2d-with-array-buffer-view.html:
  └ PASS [expected NOTRUN] Overall test
```

Reply:
it shows again in below [retry](https://community-tc.services.mozilla.com/tasks/SERV5UIyQ9iWNLyUtzgEQQ/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FSERV5UIyQ9iWNLyUtzgEQQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Ffiltered-wpt-errorsummary.log) 👀